### PR TITLE
Initial skeleton for db access

### PR DESCRIPTION
**Pull request by sebastian-mikucki**
Initial skeleton for db access

### CODE REVIEW

File: src/main/java/com/guysfromusa/carsgame/v1/MapResource.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+import static org.apache.commons.lang3.Validate.notNull;
+import static org.springframework.http.MediaType.APPLICATION_JSON_UTF8_VALUE;
+import static org.springframework.web.bind.annotation.RequestMethod.GET;
+
+/**
+ * Created by Sebastian Mikucki, 25.02.18
+ */
+@RestController
+@RequestMapping(value = "/v1/games/maps", produces = APPLICATION_JSON_UTF8_VALUE)
```
Comment (Konrad-Rys) [inline src/main/java/com/guysfromusa/carsgame/v1/MapResource.java]: Please remember to change this path

File: src/main/java/com/guysfromusa/carsgame/entities/Map.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+import static org.apache.commons.lang3.builder.ToStringStyle.SHORT_PREFIX_STYLE;
+
+@Entity
+public class Map {
```
Comment (Konrad-Rys) [inline src/main/java/com/guysfromusa/carsgame/entities/Map.java]: Map name would be misleading with object Type Map. Please consider it.

Comment (Konrad-Rys) [inline src/main/java/com/guysfromusa/carsgame/entities/Map.java]: Map name would be misleading with object Type Map. Please consider it.

Comment (sebastian-mikucki) [reply src/main/java/com/guysfromusa/carsgame/entities/Map.java]: Yes you're right, this initial class was for skeleton purpose, we can think a little bit about better name my proposal is: StreetMap, Robert will be responsible for Map API so he will use new name

**Pull request merged**